BluetoothAdvertisementsKotlin/app/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright (C) 2021 Google Inc. All Rights Reserved.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+    -->
+
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I'm not an expert in CoordinatorLayout. My understanding...
- CoordinatorLayout is a super-powered FrameLayout.
- ConstraintLayout is a super-powered ViewGroup similar to a RelativeLayout, but more flexible than RelativeLayout.

Are you using CoordinatorLayout in this fashion? Maybe you are with the app bar layout meant to be stacked on on your ConstraintLayout. If so, please ignore. This might be best practice... I've been in Wear land too long. :D

Reply:
ack